kapost_deploy.gemspec

```diff
@@ -32,6 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rubocop", "~> 0.37"
   spec.add_development_dependency "codeclimate-test-reporter"
   spec.add_development_dependency "rubygems-tasks"
+  spec.add_development_dependency "climate_control"
```

Comment:
Not sure. We don't in `arugula`. Seems safe enough since it's just a small gem for testing but I could go either way.